### ISSUE TITLE
When hiding and showing the graph, pointer events no longer work

### ISSUE THREAD

**Issue by Reldey** [vasturiano/force-graph#10]
I have been using jquery to .hide(), .empty(), and .detach() the div containing the force graph, and when I try to redraw the graph, no mouse pointer events work (on hover tooltip doesn't display, even though it's div and position show in the div) and the style=cursor:point tag isn't fired on the parent div.

**Comment by Reldey** [comment]
I realized I was updating the function out of scope, may have led to some weird behaviors. Have fixed it,
